deploy/helm/wg-access-server/values.yaml

```diff
@@ -56,6 +56,7 @@ imagePullSecrets: []
 
 image:
   repository: ghcr.io/freifunkmuc/wg-access-server
+  image: ""
```

Comment:
Does this need to be `tag`? The new code in deployment.yaml mentions `.Values.image.tag`.

Reply:
Ah yes, you're correct^^